translations/portuguese.json

```
@@ -117,7 +117,7 @@
 		"This room has slow-chat enabled. You can only talk once every ${time} seconds.":
 		"O slow-chat desta sala está ativado. Você apenas pode falar a cada ${time} segundos.",
 		"Your username contains a phrase banned by this room.": "Seu nome de usuário contém uma frase banida por esta sala.",
-		"Your status message contains a phrase banned by this room.": "A mensagem do seu status contém uma frase banida por esta sala.",
+		"Your statusMessage contains a phrase banned by this room.": "A mensagem do seu status contém uma frase banida por esta sala.",
```

```suggestion
		"Your statusMessage contains a phrase banned by this room.": "A mensagem do seu status contém uma frase banida por esta sala.",
```

I'm guessing you did a regex search/replace `status.message`? You should have just turned it to plaintext whole-word mode...

---

Ugh.

---

Fixed. My bad.